recipes/qtxlsxwriter/all/conandata.yml

```diff
@@ -0,0 +1,8 @@
+sources:
+  "cci.20200319":
+    url: "https://github.com/dbzhang800/QtXlsxWriter/archive/40610a55cb6eaf07b988d4b7af0b65e37571f83c.zip"
```

Comment:
We have a strong preference for using official releases https://github.com/dbzhang800/QtXlsxWriter/releases/tag/v0.3.0

I do not see anything important in the last 4 commits, can you please use the release

Reply:
The problem with the v0.3.0 release is that it doesn't have a license. The license was added later in [d013edc](https://github.com/dbzhang800/QtXlsxWriter/commit/d013edc9cf39b450f035528114b349a70b1507ba). I could copy the license from the master to v0.3.0, but I don't think it's a good idea.

Reply:
It's happened in the past, you can do something like this https://github.com/conan-io/conan-center-index/blob/3403cd916d2f95446ab434b2ba0197e858f9fd21/recipes/approvaltests.cpp/all/conandata.yml#L5 and it would be only for this release.

The alternative would be to manually extract the license from the `*.h` but I am not a fan of that

But lastly this could be an option too... It's a little against our guidelines but we can wait for a Conan team member to give their feedback

Reply:
The version you picked is correct.... We have a _preference_ for using official releases.
The format you are using as per the docs https://github.com/conan-io/conan-center-index/blob/master/docs/faqs.md#what-version-should-packages-use-for-libraries-without-official-releases

> The notation shown below is used for publishing packages where the original library does not make official releases. Thus, we use a format which includes the date stamp corresponding to the date of a commit: cci.<YEAR MONTH DAY>.

Reply:
@SSE4 @uilianries
Could you give your opinion, please?

Reply:
@vaerizk , please, follow @prince-chrismc recommendation. We prefer official releases. As you need a license file, follow ApprovalTest approach, which @prince-chrismc pointed. Creating a virtual version only for that file is not enough. It will be better asking to upstream tagging a new version instead.

Reply:
The author of the library could change the license after the `0.3.0` release. In that case the package would have two different licenses: the one in the header and the one taken from the master. I don't think a license change happens often, still it is possible. Fortunately, in our case both licenses are MIT, so I think there is nothing to worry about. [Done](https://github.com/conan-io/conan-center-index/pull/4838/commits/caa0e6311321874694b1981c83bb0a5f01ef32e9).